test/unit/range/container/container_test_template.hpp

```diff
@@ -20,21 +20,17 @@
 using seqan3::operator""_dna4;
 
 template <typename T>
-class container_ : public ::testing::Test
+class container_over_dna4_test : public ::testing::Test
```

Comment:
It is extremely weird, why a template is constrained on the alphabet. The alphabet should be either given explicitly or, since we require it to be a container, extracted from the given test type.

Reply:
I explicitly named the test template `over_dna4` to raise attention to this "extremely weird" test but I think refactoring the whole container test should not be part of this PR.

Reply:
Please add at least a ticket to refactor the container test. We have other ideas to also make a view test template.

Reply:
https://github.com/seqan/product_backlog/issues/150